packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -55,18 +60,23 @@ const ContractInfo = ({
 
     const basis_text = has_error_or_not_loaded
         ? stakeOrPayout()
-        : localize('{{value}}', { value: proposal_info.obj_contract_basis.text });
+        : localize('{{value}}', { value: is_turbos ? 'Payout per point' : proposal_info.obj_contract_basis.text });
```

Comment:
```suggestion
        : localize('{{value}}', { value: is_turbos ? localize('Payout per point') : proposal_info.obj_contract_basis.text });
```

Reply:
Moreover the value is a variable and it is storing the result.
then result is localized
`localize('{{value}}'`
so i dont think we need to do this.
BUt for precaution i have implemented your suggestion.